### PR TITLE
Removes the /explosive subtype.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Pretty much what the title says. Additional mapping to keep the .dmm loadable.

## Why It's Good For The Game
The path was even more useless than /device, no specific vars were defined for it, just the path itself.

### CODE REVIEW

File: _maps/WIP/Sulaco_bottom.dmm [hunk 1]
```diff
@@ -4883,8 +4882,20 @@
...
 /area/sulaco/telecomms)
 "kI" = (
-/turf/open/floor/engine/vacuum,
-/area/sulaco/telecomms)
+/obj/structure/cable{
+	d1 = 4;
+	d2 = 8;
+	icon_state = "4-8"
+	},
+/obj/machinery/atmospherics/pipe/simple/yellow/hidden{
```
Comment (Rohesie) [inline _maps/WIP/Sulaco_bottom.dmm]: Are these map changes intended?

File: _maps/WIP/Z.01.Desert-Dam.dmm [hunk 2]
```diff
@@ -3538,8 +3528,8 @@
 /area/desert_dam/interior/lab_northeast/biology)
 "alh" = (
 /obj/structure/table/reinforced,
-/obj/item/device/defibrillator,
-/obj/item/device/defibrillator,
+/obj/item/defibrillator,
+/obj/item/defibrillator,
```
Comment (Rohesie) [inline _maps/WIP/Z.01.Desert-Dam.dmm]: Did La's PR  not properly remove the `device` paths from the maps?

Comment (Ghommie) [reply _maps/WIP/Z.01.Desert-Dam.dmm]: they didn't remove it from the desert dam, but did for the rest.

File: _maps/WIP/Sulaco_bottom.dmm (hunk 1 above)
Comment (Ghommie) [reply _maps/WIP/Sulaco_bottom.dmm]: Probably, I had to remove a couple PDA related items that LaKiller didn't from the map. And the .dmm probably reorganized itself, as result.

**Comment by spookydonut** [comment]
you're undoing phil's work i believe.

**Comment by Ghommie** [comment]
> 
> 
> you're undoing phil's work i believe.

I don't believe this path has been used in any statement outside categorizing grenades, mines and plastique, so I don't quite see a problem. The subfolder is kept around for indexing repository though.

File: _maps/WIP/Sulaco_top.dmm [hunk 3]
```diff
@@ -8213,17 +8221,14 @@
 /turf/open/floor,
 /area/sulaco/cargo/office)
 "qy" = (
-/obj/item/radio/intercom{
-	anyai = 1;
-	freerange = 1;
-	name = "General Listening Channel";
-	pixel_y = 30
+/obj/machinery/atmospherics/pipe/simple/yellow/hidden{
```
Comment (DominikPanic) [inline _maps/WIP/Sulaco_top.dmm]: Don't mapmerge with changes like this, simply just mass replace the map file like it was a .dm file

**Comment by spookydonut** [comment]
I think this should be closed in favour of consolidating proc behaviour on the explosive type path

**Comment by Ghommie** [comment]
> 
> 
> I think this should be closed in favour of consolidating proc behaviour on the explosive type path

That's also a fine alternative. Might give it a look myself if no one will later on. Consider this closed until then.

**Comment by Rohesie** [comment]
This would work better done upstream, honestly.

**Pull request closed without merging**